cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -15,16 +15,16 @@ persistentVolume:
 PodManagementPolicy: Parallel
 
 Services:
-  - name: "yb-masters"
-    label: "yb-master"
+  - name: "masters"
+    label: "master"
```

Comment:
Why drop the yb prefix? Our docs for kubernetes will be out of sync with what the helm chart does.